docs/networks/mainnet/lukso-mainnet.md

```diff
@@ -0,0 +1,228 @@
+# Running a LUKSO node
+
+To be part of LUKSO's network as a validator one should understand the concept of staking.
+Staking is the process of creating and maintaining one (or more) of these validators on the to help the network maintain the consistency and security of the LUKSO blockchain.
+
+To earn staking rewards for the LUKSO Network, the job is straightforward, but crucially important: run validators with the highest quality possible, and maximize staking rewards.
+
+Node operators are responsible for:
+
+- Setting up a computer (either physical or virtual)
+- Configuring it correctly, including their home network if applicable
+- Installing the LUKSO cli to perform validation
+- Securing it, both from outside and inside threats
+- Maintaining it for the life of their validators
+
+It's a big responsibility, and not a simple set-it-and-forget-it kind of job; you need to care for your node for as long as it's staking.
+
+## A Proof of Stake Chain
+
+<--! TODO: Hugo/Fabian, please review this text and define the LYX quantity to be deposited to run a node , or point to the community/governance discussion about this amount-->
```

Comment:
According to here, https://medium.com/lukso/lukso-mainnet-timeline-and-process-dd997fe811c8

it will be 32LYXe

Reply:
fixed